libs/io/writer.py

```
@@ -1,4 +1,5 @@
-from awsglue.context import GlueContext, DynamicFrame
+from awsglue.context import DynamicFrame, GlueContext
```

Ruff (0.12.2)

`awsglue.context.GlueContext` imported but unused

Remove unused import: `awsglue.context.GlueContext`

(F401)